packages/storage-documents/src/scan-result-types/scan-result.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+export type ScanResultState = 'pass' | 'fail';
```

Comment:
Removed the `pending` state here because incomplete scans will not have a scanResult